arm/Microsoft.Compute/virtualMachines/.parameters/parameters.json

```diff
@@ -114,10 +114,10 @@
             }
```

Comment:
Please update `KeyEncryptionKeyURL` above to `https://adp-sxx-az-kv-x-001.vault.azure.net/keys/keyEncryptionKey/c263cf982a8442f69c3a0ece5e345b6c`